flash_image/recovery_mode.rst

```diff
@@ -3,4 +3,52 @@
 How to enter recovery mode
 ##########################
 
-TODO
+1. Connect power cable to ROScube and power on it.
+--------------------------------------------------
+
+.. image:: images/prepare-host-computer.png
+  :width: 80%
+  :align: center
+
+.. image:: images/power-on.png
+  :width: 80%
+  :align: center
+
+2. Make sure Micro USB cable is in good quality.
+------------------------------------------------
+
+.. image:: images/usb-in-good-quality.png
+  :width: 80%
+  :align: center
+
+
+.. _recovery mode:
+
+3. Reset to recovery mode.
+--------------------------
+
+i) Steady pressed RECOVERY button.
+
+.. image:: images/steady-press-recovery-button.png
+  :width: 80%
+  :align: center
+
+ii) Hold RECOVERY button pressed and then press RESET button. (The button next to RECOVERY button)
+
+.. image:: images/press-reset-button.png
+  :width: 80%
+  :align: center
+
+iii) Release RESET button dirst and then release RECOVERY button.
+
+.. image:: images/release-reset-button.png
+  :width: 80%
+  :align: center
+
+4. Now ROScube is in recovery mode now.
+---------------------------------------
+
+.. image:: images/nvidia-usb.png
```

Comment:
Your image should not contain "ROS 1 noetic"

Reply:
I update a new image.